README.md

```diff
@@ -214,6 +214,15 @@ models:
 
 ---
 ### SQL helpers
+#### fetch ([source](macros/sql/fetch.sql))
+This macro returns a dictionary from a sql query.
+
+Usage:
+```
+-- Returns a dictionary of the users table where the state is California
+{% set ___ = dbt_utils.fetch("select * from ref{{'users'}} where state = 'CA' ") %}
```

Comment:
It's a good idea to make usage docs like this directly usable. Can you replace `___` here with `users`? I also think this is a syntax error - it should be something like:
```
{% set users = dbt_utils.fetch("select * from " ~ ref('users') ~ " where state = 'CA' ") %}
```